README.md

```diff
@@ -104,48 +111,54 @@ Codename:        bionic
 ```
 
 ## Stop an instance to save resources
+
 ```
 $ multipass stop dancing-chipmunk
 ```
 
-## Delete the instance
+## Delete an instance
+
 ```
 $ multipass delete dancing-chipmunk
 ```
 
-It will now show up as deleted:
+The instance will now show up as deleted:
+
 ```$ multipass list
+
 Name                    State             IPv4             Release
 snapcraft-asciinema     STOPPED           --               Ubuntu Snapcraft builder for Core 18
 dancing-chipmunk        DELETED           --               Not Available
 ```
 
-And when you want to completely get rid of it:
+If you want to completely get rid of it:
 
 ```
 $ multipass purge
 ```
 
 ## Get help
-```
 
+```
 multipass help
 multipass help <command>
 ```
 
-# Get involved!
+# Make your own build
+
```

Comment:
`Make your own build` sounds weird, what about `Developing Multipass` like [gtest](https://github.com/google/googletest/blob/main/CONTRIBUTING.md#developing-google-test-and-google-mock)

Reply:
I like this, @georgeliao ! Thanks for the suggestion. I'll use "Start developing Multipass".